README.md

```diff
@@ -1 +1,189 @@
-# Webpack-BoilerPlate
\ No newline at end of file
+![](https://img.shields.io/badge/MazeFlix-red)
+
+# MazeFlix
+
+> MazeFlix is an entertainment hub where you can find the latest and your favorite TV shows. You can view the details of the show, like a show or even comment. You can also see other comments which might give you a general idea about what people think about the show.
+> It is built with HTML, CSS and JavaScript with data from the TvMaze API(tv show data) and the Involvement API(likes and comments)
+
+## Project Presentation
+[Presentation Link](https://drive.google.com/drive/folders/1F7nq4FhOPCONHq84RJJ6RLaSfYrdU53a)
+
```

Comment:
- [ ] You have really done a good presentation but one thing missing. Please highlight a piece of the code you find interesting or are proud of making.